amundsen_application/static/js/components/common/ResourceListItem/DashboardListItem/index.tsx

```diff
@@ -13,7 +12,7 @@ import { formatDate } from 'utils/dateUtils';
 
 import { ResourceType, DashboardResource } from 'interfaces';
 
-import { NO_TIMESTAMP_TEXT } from 'components/constants';
+import { NO_TIMESTAMP_TEXT } from '../../../../constants';
```

Comment:
why are we opting to reference paths in this way?

Reply:
Because we don't have an "alias" for the root of the "js" I am afraid!